torch/utils/flop_counter.py

```diff
@@ -293,6 +478,10 @@ def sdpa_backward_flop(grad_out_shape, query_shape, key_shape, value_shape, *arg
     aten._scaled_dot_product_flash_attention: sdpa_flop,
     aten._scaled_dot_product_efficient_attention_backward: sdpa_backward_flop,
     aten._scaled_dot_product_flash_attention_backward: sdpa_backward_flop,
+    aten._flash_attention_forward: _flash_attention_forward_flop,
```

Comment:
nit: might be worth a comment mentioning that the calculation of flops for the lower ops are based on the assumption that NJTs are being passed to them. It's possible that some other tensor subclass overrides SDPA later on with its own dispatching logic that calls the lower ops, and the calculation could be wrong for that case